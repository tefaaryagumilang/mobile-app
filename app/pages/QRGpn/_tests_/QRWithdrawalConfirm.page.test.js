import React from 'react';
import renderer from 'react-test-renderer';
import QRWithdrawalConfirm from '../QRWithdrawalConfirm.page';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';

const store = createStore(() => ({}));
const DecoratedForm = reduxForm({form: 'QRForm'})(QRWithdrawalConfirm);

describe('Home: QRWithdrawalConfirm page', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Provider store={store}><DecoratedForm/></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
