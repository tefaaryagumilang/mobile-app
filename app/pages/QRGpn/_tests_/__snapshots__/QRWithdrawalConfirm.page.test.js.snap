// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home: QRWithdrawalConfirm page renders correctly 1`] = `
<KeyboardAwareScrollView
  contentContainerStyle={
    Array [
      Object {
        "backgroundColor": "#F1F3F6",
        "flexGrow": 1,
        "justifyContent": "space-between",
      },
    ]
  }
  extraHeight={200}
  keyboardShouldPersistTaps="handled"
  style={
    Array [
      Object {
        "padding": 20,
      },
      Object {
        "marginTop": 667,
      },
    ]
  }
>
  <View>
    <View>
      <Text>
        Confirm QR Withdrawal
      </Text>
    </View>
  </View>
  <View>
    <View>
      <Text>
        From Account
      </Text>
      <Text>
        
      </Text>
    </View>
    <View>
      <Text>
        Amount
      </Text>
      <Text>
        
      </Text>
    </View>
    <View>
      <Text>
        Note
      </Text>
      <Text>
        
      </Text>
    </View>
  </View>
  <View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#FA3F70",
                "borderRadius": 30,
                "flexDirection": "row",
                "height": 47,
                "justifyContent": "center",
                "paddingHorizontal": 15,
              },
              undefined,
              Object {},
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontWeight": "bold",
              }
            }
          >
            LANJUTKAN
          </Text>
        </View>
      </View>
    </View>
  </View>
</KeyboardAwareScrollView>
`;
